api/types/session.go

```diff
@@ -224,10 +224,12 @@ func (ws *WebSessionV2) CheckAndSetDefaults() error {
 	if err := ws.Metadata.CheckAndSetDefaults(); err != nil {
 		return trace.Wrap(err)
 	}
-
 	if ws.Spec.User == "" {
 		return trace.BadParameter("missing User")
 	}
+	if ws.Spec.LoginTime.IsZero() {
+		ws.Spec.LoginTime = time.Now().UTC()
+	}
```

Comment:
nit: I'm not sure about setting login time for existing sessions.  if the correct time wasn't recorded when the session was created, might it be better to leave it blank rather than present an incorrect value?  I don't feel strongly either way, but it feels a bit odd.

Reply:
Ok, that's fair. I was questionable on this one too. Will remove.